### ISSUE TITLE
Clean up compile warnings within the UPP code

### ISSUE THREAD

**Issue by MatthewPyle-NOAA** [NOAA-EMC/UPP#964]
The UPP to be used for RRFS creates a few warning prints, seemingly in stub codes related to WRF I/O.  
```
/lfs/h2/emc/lam/noscrub/Matthew.Pyle/rrfs-workflow_warnings/sorc/UPP/sorc/ncep_post.fd/io_int_stubs.f(4): warning #6843: A dummy argument with an explicit INTENT(OUT) declaration is not given an explicit value.   [STATUS]
SUBROUTINE ext_int_get_dom_ti_real ( DataHandle,Element,   Data, Count, Outcount, Status )
----------------------------------------------------------------------------------^
/lfs/h2/emc/lam/noscrub/Matthew.Pyle/rrfs-workflow_warnings/sorc/UPP/sorc/ncep_post.fd/io_int_stubs.f(4): warning #6843: A dummy argument with an explicit INTENT(OUT) declaration is not given an explicit value.   [DATA]
SUBROUTINE ext_int_get_dom_ti_real ( DataHandle,Element,   Data, Count, Outcount, Status )
-----------------------------------------------------------^
/lfs/h2/emc/lam/noscrub/Matthew.Pyle/rrfs-workflow_warnings/sorc/UPP/sorc/ncep_post.fd/io_int_stubs.f(4): warning #6843: A dummy argument with an explicit INTENT(OUT) declaration is not given an explicit value.   [OUTCOUNT]
SUBROUTINE ext_int_get_dom_ti_real ( DataHandle,Element,   Data, Count, Outcount, Status )
```
I did a quick test removing these lines from io_int_stubs.f, and it compiled cleanly (but didn't try running with the built executable).  So might be an easy fix.  

 ```
!--- get_dom_ti_real
 SUBROUTINE ext_int_get_dom_ti_real ( DataHandle,Element,   Data, Count, Outcount, Status )
-  INTEGER ,       INTENT(IN)  :: DataHandle
-  CHARACTER*(*) :: Element
-  REAL ,          INTENT(OUT) :: Data(*)
-  INTEGER ,       INTENT(IN)  :: Count
-  INTEGER ,       INTENT(OUT) :: Outcount
-  INTEGER ,       INTENT(OUT) :: Status

```

**Comment by WenMeng-NOAA** [comment]
@MatthewPyle-NOAA Could you submit a PR to the branch 'release/rrfs_v1'? Thanks!

**Comment by MatthewPyle-NOAA** [comment]
https://github.com/NOAA-EMC/UPP/pull/967

**Comment by WenMeng-NOAA** [comment]
This issue was solved in the PR #967.
